instaloader/instaloader.py

```diff
@@ -563,23 +563,20 @@ def download_post(self, post: Post, target: Union[str, Path]) -> bool:
                         post.get_sidecar_nodes(self.slide_start, self.slide_end),
                         start=post.mediacount if self.slide_start < 0 else self.slide_start + 1
                 ):
+                    suffix = str(edge_number)
+                    if '{filename}' in self.filename_pattern:
+                        suffix = ''
```

Comment:
I think this should be:

```suggestion
                    suffix = str(edge_number)  # type: Optional[str]
                    if '{filename}' in self.filename_pattern:
                        suffix = None
```

Because if suffix is not None, download_pic will add `_` to the filename:

https://github.com/instaloader/instaloader/blob/828fa71edee9cbe82b00fe2591a6691b9000b3cc/instaloader/instaloader.py#L282-L283

Reply:
correct